google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -923,6 +938,12 @@ public boolean isAutoConfigEmulator() {
     return autoConfigEmulator;
   }
 
+  /** The initial rpcPriority value for connections created by this {@link ConnectionOptions} */
+  /** The {@link Options.RpcPriority} to use for the connection. */
```

Comment:
nit: Don't use two separate javadoc blocks for a single method, merge them into one.